.github/workflows/code-quality.yml

```diff
@@ -1,7 +1,7 @@
 name: "Check code format and quality"
 on:
   push:
-    branches: [master]
+    branches: [develop]
```

Comment:
If we can still push changes to the `master` branch then this should  be applied on both branches